peer.go

```diff
@@ -463,6 +463,8 @@ func (p *peer) loadActiveChannels(chans []*channeldb.OpenChannel) (
 			fallthrough
 		case dbChan.HasChanStatus(channeldb.ChanStatusCommitBroadcasted):
 			fallthrough
+		case dbChan.HasChanStatus(channeldb.ChanStatusCoopBroadcasted):
```

Comment:
If a peer shuts down during the coop close negotiation, then we won't be able to retry another coop close, which seems undesired?

Reply:
hmm that's true, hadn't thought about that. i have another follow up in this area where it'd be easy to change, for now tho it prevents the itests from flaking.